.github/workflows/update-docs.yml

```diff
@@ -143,7 +145,7 @@ jobs:
           prompt_file: docs-repo/analysis_prompt.md
           anthropic_api_key: ${{ secrets.ANTHROPIC_API_KEY }}
           allowed_tools: "Bash,Read,Write,Edit,MultiEdit,Glob,Grep,LS,Task,WebFetch,WebSearch,TodoRead,TodoWrite,NotebookRead,NotebookEdit"
-          max_turns: 15
+          max_turns: 40
```

Comment:
I think this the the change that enabled it to finish the workflow and open the doc PR